dotCMS/src/main/java/com/dotcms/util/CollectionsUtils.java

```diff
@@ -501,10 +503,65 @@
     } // map.
 
     /**
+     * Creates a new map based on a list of key/value parameters.
      * 
-     * @param entries
+     * @param key1
+     * @param value1
+     * @param key2
+     * @param value2
+     * @param key3
+     * @param value3
+     * @param key4
+     * @param value4
+     * @param key5
+     * @param value5
+     * @param key6
+     * @param value6
+     * @param key7
+     * @param value7
+     * @param key8
+     * @param value8
+     * @param key9
+     * @param value9
+     * @param key10
+     * @param value10
+     * @param key11
+     * @param value11
+     * @param key12
+     * @param value12
+     * @param key13
+     * @param value13
+     * @param key14
+     * @param value14
+     * @param key15
+     * @param value15
+     * @param key16
+     * @param value16
+     * @param key17
+     * @param value17
+     * @param key18
+     * @param value18
      * @return
      */
+    public static <K, V> Map<K, V> map(final K key1, final V value1, final K key2, final V value2, final K key3, final V value3,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Method has 36 parameters, which is greater than 7 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00107)